docs/whatsnew/6.0.rst

```diff
@@ -14,32 +14,60 @@ On this page, you can read about some of the big changes in this release.
 
 ``sunpy`` 6.0 also includes a large number of smaller improvements and bug fixes, which are described in the :ref:`changelog`.
 
-PUT STATS HERE
+**PUT STATS HERE**
+
+Updates to minimum dependencies
+===============================
+
+The minimum required version of Python has been updated to 3.10.
+
+The minimum required versions of core dependencies have been updated:
+
+- astropy >= 5.2.0
+
+The minimum required versions of these optional dependencies has also been updated:
+
+- asdf-astropy >= 0.2.0
+- asdf >= 2.11.0
+- dask >= 2022.5.2
+- scikit-image >= 0.19.0
+- scipy >= 1.8.0
+- lxml >= 4.9.0
+- reproject >= 0.9.0
+- beautifulsoup4 >= 4.11.0
+- tqdm >= 4.64.0
+- zeep >= 4.1.0
+- spiceypy >= 5.0.0
+- cdflib >= 0.4.4
+- h5netcdf >= 1.0.0
+- h5py >= 3.7.0
+- pandas >= 1.4.0
+- opencv-python >= 4.6.0.66
 
 Removal of ``sunpy.database``
------------------------------
+=============================
 
 The ``sunpy.database`` module has not been actively maintained for over a year now and has a number of outstanding issues.
 It has been deprecated since sunpy 4.1, and since there have been no major objections from the communityu since then we have completely removed ``sunpy.database`` in sunpy 6.0.
```

Comment:
```suggestion
It has been deprecated since sunpy 4.1, and since there have been no major objections from the community since then we have completely removed ``sunpy.database`` in sunpy 6.0.
```